emulators/qemu/Portfile

```diff
@@ -8,7 +8,7 @@ PortGroup legacysupport 1.1
 legacysupport.newest_darwin_requires_legacy 15
 
 name                    qemu
-version                 6.2.0
+version                 7.0.0
 revision                1
```

Comment:
`revision` needs to reset to 0 on version bump.

Reply:
Hi @herbygillot,

Sorry I missed resetting the revision to 0.  I have updated my commit to fix this.

Thanks,

-ranga